pgml-admin/pgml_admin/settings.py

```diff
@@ -91,7 +92,7 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "OPTIONS": {"options": "-c search_path=pgml"},
```

Comment:
This is something I worry slightly about. I'd rather be explicit in every query that reaches out to the host schema's from admin. For example, imagine we use the django sessions backed by the database, and the host db has their own db backed sessions table. If they forget to run migrations, we may read the wrong sessions table in admin. I would rather fail with a table not found error.

Reply:
Good point. I moved the Django tables into the `pgml` schema:

```
pgml_development=# select count(1) from pgml.django_migrations;
 count 
-------
    19
(1 row)
```